### DISCUSSION TITLE
Ability to retrieve prev_end_date_success

### DISCUSSION THREAD

**Discussion by vuphamcs** [apache/airflow#33345] [General]
### Description

Is there a variable similar to `prev_start_date_success` but for the previous DAG run’s completion date? The value I’m hoping to retrieve to use within the next DAG run is `2023-08-10 16:04:30`

![image](https://github.com/apache/airflow/assets/1600760/9e2af349-4c3e-47e1-8a1a-9d8827b56f57)

### Use case/motivation

One particular use case is to help guarantee that the next DAG run only queries data that was inserted during the existing DAG run, and not the past DAG run.

```python
prev_ts = context['prev_end_date_success']

sql = f"SELECT * FROM results WHERE created_at > {context['prev_end_date_success']}"
```

### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by rohan472000** [comment]
@vuphamcs Are you trying to find a variable that provides the completion date of the previous DAG run for use in your current run?

**Comment by potiuk** [comment]
Questions like this should be discussions. not issues (it's explained when you open issue - in the issue template). Keep it in mind for the future. Converted to a discussion.

**Reply by RNHTTR** [reply]
That's my bad; I encouraged @vuphamcs to open an Issue. Given `prev_start_date_success` exists, it seems reasonable that `prev_end_date_success` should exist as well IMO.

**Reply by potiuk** [reply]
Cool. Yeah - I saw a question marjk. And given the (ltotle) complexity - sure  @vuphamcs -> PRs are most welcome to add  those.

**Reply by potiuk** [reply]
Recreated an issue from it and added "good first issue" label.
